src/usr/local/pkg/miniupnpd.xml

```diff
@@ -286,9 +295,9 @@
 				criteria. The entries only control access for IPv4 clients, they do not apply
 				to IPv6 clients.
 				<br/><br/>
-				Format: <tt>[allow or deny] [ext port or range] [int ipaddr or ipaddr/CIDR] [int port or range]</tt>
+				Format: <tt>(allow|deny) (ext port|range) (int IP|IP/netmask) (int port|range)</tt>
 				<br/>
-				Example: <tt>allow 1024-65535 192.168.0.0/24 1024-65535</tt>
+				Example: <tt>allow 1024-65535 192.168.1.0/24 1024-65535</tt>
```

Comment:
The current text is fine since this is simply an example and isn't meant to follow any system defaults.

Reply:
OK.